### DISCUSSION TITLE
What is the Docusaurus v2 release philosophy?

### DISCUSSION THREAD

**Discussion by HonkingGoose** [facebook/docusaurus#4206] [Q&A] [answered]
I'm seeing all kinds of cool things land in the main repository (dependency updates, new features and fixes), however the last Docusaurus Alpha release is `2.0.0-alpha.70` which released on December 17th 2020.

I'm wondering, when can I see these nice things in a stable release?
Is there some kind of schedule/criteria that determines when a new release is made?
I couldn't find any docs about your release process, so that's why I'm asking here.

Also what's the thinking behind the `canary` release channel, is that intended for power-users who want the latest stuff right-now?

I hope I'm not being too impatient here, if I am, just tell me to sit tight and wait... 😉

**Comment by slorber** [comment, marked as answer]
Hi,

There's no particular rule, but the next release is a big one containing the official i18n support + doc (even if it's somehow already released).
I want the Jest site on v2 with i18n to be online for the release and finishing the last details to make this happen.

The `@canary` tag is for all code on master, so yeah it's for early adopters, but it should be expected to be stable otherwise we wouldn't merge it in the first place. If you fix your dep to a specific canary release version at a given time that's not so risky to use. I would advise to use it anyway because when we release it's kind of what you are going to get 😅

**Reply by HonkingGoose** [reply]
Thanks for responding to my questions! 😄
